src/install/polymc.md

```diff
@@ -31,16 +27,20 @@ mods and get playing.
 
 To install mods to your PolyMC instance, simply follow these steps:
 
-1. Right-click on the instance you'd like to install mods to and select **Edit Instance**, or click on it once and 
-   select **Edit Instance** from the menu on the right of the window
-2. Select **Mods** from the list on the left, and click on the **Download mods** button in the menu on the right side
-   of the window
-3. Find a Fabric or Quilt mod from CurseForge or Modrinth, and click the **OK** button
-4. Ensure you have the **Quilt Standard Libraries** installed, which you can find
-   [on Modrinth](https://modrinth.com/mod/qsl)
+1. Right-click on the instance you'd like to install mods to and select **Edit Instance**, or click on it once and select **Edit Instance** from the menu on the right of the window
+2. Select **Mods** from the list on the left, and click on the **Download mods** button in the menu on the right side of the window
+3. Find a mod from CurseForge or Modrinth, and click the **Select mod for download** button
+4. Ensure you have selected **Quilt Standard Libraries** to be installed with the mod downloader, alternatively download [on Modrinth](https://modrinth.com/mod/qsl)
```

Comment:
```suggestion
4. Ensure you have selected **Quilted Fabric API** to be installed with the mod downloader, alternatively download [on Modrinth](https://modrinth.com/mod/qsl)
```

Reply:
Right I'll ammend this to be up to date, im pretty sure the other pages also refer to qsl so I might add that into the pr.